src/javaharness/java/arcs/crdt/CrdtTestHelper.java

```diff
@@ -0,0 +1,109 @@
+package arcs.crdt;
+
+import java.util.Objects;
+
+@SuppressWarnings("unchecked")
+public class CrdtTestHelper {
+  public static class Data implements Referenceable {
+    public final String id;
+
+    public Data(String id) {
+      this.id = id;
+    }
+
+    @Override
+    public String getId() {
+      return id;
+    }
+
+    @Override
+    public boolean equals(Object other) {
+      return this.id.equals(((Data) other).id);
+    }
+
+    @Override
+    public int hashCode() {
+      return this.id.hashCode();
+    }
+  }
+
+  public static void testMergeModels() {
```

Comment:
See other comment. This file probably shouldn't exist (or possibly it should, but the test methods should belong in the test file).

If you can't put the CrdtTestHelper.Data class in the same test file, you can leave this as a test helper class, but it should ideally live inside a .testing subpackage

Reply:
deleted this file